app/scripts/lib/account-tracker.test.js

```diff
@@ -146,12 +148,12 @@ describe('Account Tracker', () => {
       expect(blockTrackerStub.removeListener).toHaveBeenNthCalledWith(
         1,
         'latest',
-        accountTracker._updateForBlock,
+        expect.any(Function),
```

Comment:
Is it possible to make keep this more specific?

Reply:
I'm not sure how as the `_updateForBlock` method is now private and inaccessible from the test